spec/evaluator_spec.rb

```diff
@@ -1,4 +1,5 @@
 require 'spec_helper'
+require 'dentaku/token_matcher'
```

Comment:
This is necessary for `rspec spec/evaluator_spec.rb`, otherwise it complains that it doesn't know what `Token` is.